pkg/phlaredb/symdb/resolver.go

```diff
@@ -295,3 +306,18 @@ func (r *Symbols) Tree(
 ) (*model.Tree, error) {
 	return buildTree(ctx, r, appender, maxNodes)
 }
+
+func (r *Symbols) SetSymbolizer(sym *symbolizer.Symbolizer) {
+	r.Symbolizer = sym
+}
+
+func (r *Symbols) needsDebuginfodSymbolization(loc *schemav1.InMemoryLocation, mapping *schemav1.InMemoryMapping) bool {
+	if r.Symbolizer == nil {
+		return false
+	}
+	if len(loc.Line) == 0 {
```

Comment:
I guess it's time to move this logic (unsymbolized fallback) https://github.com/grafana/pyroscope/blob/466aaa8194e06aab452b00f56efd5ddd8bcff311/pkg/ingester/otlp/convert.go#L166 somewhere here to Symbols/resolver

Reply:
This was changed in https://github.com/grafana/pyroscope/pull/3792.